ADR/0026-appstudio-object-labels.md

```diff
@@ -0,0 +1,36 @@
+# 26. Record architecture decisions
```

Comment:
Can you provide a more descriptive title?

Reply:
Yep, I can :) Was so glad I finally put the ADR together that I forgot about the most important thing :) It is now updated.